# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/mozilla/tests/selection/cross-shadow-boundary-slot-complex-3.html
# Commit: 4f30d7afdf22
# Full Hash: 4f30d7afdf22c652481513564b60b37053c0948b
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-05-08 04:32:48
# Regressor Bug: 1932150
# File Overlap Count: 1
# Description:
#   Revert "Bug 1932150 - Fix build bustage" for causing build bustages.
#   
#   This reverts commit 0681b181d6a8a48ac460a97dacb7fca91f9989c3.
#   
#   Revert "Bug 1932150 - Fix lint failure" for causing Build bustage.
# ==============================================================================

diff -r 1e64a48e3517 -r 4f30d7afdf22 testing/web-platform/mozilla/tests/selection/cross-shadow-boundary-slot-complex-3.html
--- a/testing/web-platform/mozilla/tests/selection/cross-shadow-boundary-slot-complex-3.html	Wed May 07 20:11:46 2025 +0300
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,28 +0,0 @@
-<!doctype html>
-<head>
-<link rel="match" href="cross-shadow-boundary-slot-complex-3-ref.html"/>
-</head>
-<div>
-  <template shadowrootmode="open">
-    <slot></slot>
-  </template>
-  <div id="nested">
-    <template shadowrootmode="open">
-      <span id="start">Start</span>
-      <slot></slot>
-      <span>End</span>
-    </template>
-    <span>Another</span>
-  </div>
-  <span>Second</span>
-  <span id="end">Third</span>
-</div>
-<script>
-  // art Another End
-  // Second Thir
-  window.getSelection().setBaseAndExtent(
-    nested.shadowRoot.getElementById("start").firstChild,
-    2,
-    end.firstChild,
-    4);
-</script>